src/resolvers.ts

```diff
@@ -16,6 +17,16 @@ import {
 
 const resolvers = {
     Query: {
+        degrees: () => {
+            const matchingDegrees = getDegrees();
+
+            return matchingDegrees;
+        },
+        degree: (_parent: Degree, args: { id: string; }) => {
+            const matchingDegree = getDegreeById(args.id);
```

Comment:
separate ticket: I noticed we aren't `await` ing any of our queries yet. (not just these new ones), we should update them now that we've successfully connected the DB to the graph!

Reply:
I was matching the style of the current code. Do you feel `await` is better? Especially since this file is small, perhaps you could refactor it